test/e2e/utils/recordedit-helpers.js

```diff
@@ -7,40 +7,44 @@ var EC = protractor.ExpectedConditions;
 
 //test params for markdownPreview
 var markdownTestParams = [{
```

Comment:
Yes, the module `testPresentationAndBasicValidation` is called from both **add** and **edit** spec files in _recordedit_. This module contains most of the code in `recordedit-helpers.js` including  that for markdown test.